sprintQuovantis/sprintParam/views.py

```diff
@@ -103,3 +106,76 @@ def post(self, request):
                     "message": "Data not stored"
                 },
                 status=status.HTTP_400_BAD_REQUEST)
+
+
+class UserVote(APIView):
+    """
+    This class is creating for vote records
+    """
+    def post(self, request):
+        """
+        this method is created for inserting the voting data
+        :param request: format of the request
+        :return: Response
+        """
+        try:
+            serializer = VoteSerializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
+            serializer.save()
+            return Response(
+                {
+                    "Message": "User Vote Successfully",
+                    "data": serializer.data
+                },
+                status=status.HTTP_201_CREATED
+            )
+        except ValidationError:
+            logging.error("Validation Failed")
+            return Response(
+                {
+                    "Message": "Validation Failed"
+                },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        except Exception as e:
+            logging.error(e)
+            return Response(
+                {
+                    "Message": "Invalid User"
+                },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+
+class Result(APIView):
+    """
+    This class Using for Voting Result
+    """
+
+    def get(self, request):
+        """
+        this method is created for fetching the voting data
+        :param request: format of the request
+        :return: Response
+        """
+        try:
+            votes = Votes.objects.filter(vote_for=request.data.get("vote_for"))
```

Comment:
`request.data.get('vote_for')`

Where is vote_for coming from? request body? If yes, GET requests should never have a request body, only url path params or query params.

And result api should return who got the most votes in a particular sprint. Please modify your query for that.